### ISSUE TITLE
Pulsar manager failed to run in 1.2.1

### ISSUE THREAD

**Issue by jonaskint** [streamnative/charts#120]
Hi, i am trying to enable pulsar-manager in the 1.2.1 version of the chart. When doing this, i am running into an issue where --redirect.host flag is failing.

```
08:33:07.878 [main] ERROR org.springframework.boot.SpringApplication - Application run failed
java.lang.IllegalArgumentException: Invalid argument syntax: --redirect.host=
	at org.springframework.core.env.SimpleCommandLineArgsParser.parse(SimpleCommandLineArgsParser.java:75)
	at org.springframework.core.env.SimpleCommandLinePropertySource.<init>(SimpleCommandLinePropertySource.java:90)
	at org.springframework.boot.DefaultApplicationArguments$Source.<init>(DefaultApplicationArguments.java:76)
	at org.springframework.boot.DefaultApplicationArguments.<init>(DefaultApplicationArguments.java:42)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1255)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1243)
	at org.apache.pulsar.manager.PulsarManagerApplication.main(PulsarManagerApplication.java:27)
```

I am not using tls, and just enabling the pulsar manager so i think the issue lies on this line: https://github.com/streamnative/charts/blob/master/charts/pulsar/templates/pulsar-manager/pulsar-manager-configmap.yaml#L85

**Update**
current config in my values file:
```yaml
pulsar_manager:
  component: pulsar-manager
  ports:
    frontend: 9527
    backend: 7750
  replicaCount: 1
  superuser:
    user: "zyx"
    password: "xyz"
    description: "Pulsar Manager Admin"
    email: support@pulsar.io
  redirect:
    host: localhost
  scripts:
    backend_entrypoint:
      command: /pulsar-manager/pulsar-manager/bin/pulsar-manager
```

But if i check my configmap i see the following:
```yaml
apiVersion: v1
data:
  backend_entrypoint.sh: |
    /pulsar-manager/pulsar-manager/bin/pulsar-manager \
      --spring.datasource.initialization-mode=never \
      --spring.datasource.driver-class-name=org.postgresql.Driver \
      --spring.datasource.url=jdbc:postgresql://127.0.0.1:5432/pulsar_manager \
      --spring.datasource.username=pulsar \
      --spring.datasource.password=pulsar \
      --pagehelper.helperDialect=postgresql \
      --backend.jwt.token="$(cat /pulsar/tokens/pulsar_manager/token)" \
      --jwt.broker.token.mode=PRIVATE \
      --jwt.broker.public.key=file:///pulsar/keys/token/public.key \
      --jwt.broker.private.key=file:///pulsar/keys/token/private.key \
      --bookie.host="http://pulsar-bookie:8000" \
      --bookie.enable=true \
      --redirect.scheme=http \
      --redirect.port=80 \
      --redirect.host= \
...
```

For some reason, the redirect.host is not filled in correctly.

**Comment by jonaskint** [comment]
I found the issue. By default control_center is enabled which causes it to use https://github.com/streamnative/charts/blob/master/charts/pulsar/templates/pulsar-manager/pulsar-manager-configmap.yaml#L78 which was empty in my case.

Closing!

**Comment by sijie** [comment]
@jonaskint Yeah. We have fixed it in 1.2.2 release as well. You can try the 1.2.2 release.
